### ISSUE TITLE
Inquiry Regarding Model and Training Hyperparameters

### ISSUE THREAD

**Issue by ww20hust** [lumimim/NeuPRINT#1]
Hello！
1. How should I set the parameters contained in the following dictionary to reproduce the 0.807 accuracy reported in the "Individual with Population+NeuPRINT+MLP" condition? 
2. Under these parameter settings, how much GPU memory would be occupied, and approximately how long did it take to complete the computation using the V100 you utilized? 
Thank you！
hparams_dict = {
          'recon_model_type': 'transformer',
          'classify_model_type': 'mlp',
          'output_distribution': 'mse',
          'use_population': True,
          'use_neighbor': True,
          'return_last': False,
          'all_sessions_neuron_count': int(all_sessions_neuron_count),
          'num_batch': 100,
          'batch_size': 1024,
          'record_frac': 0.9,
          'post_hoc_eval_epoch_freq': 10,
          'num_sample_neurons': 512,
          'num_epochs': 400,
          'num_post_hoc_epochs': 2000, # default 5000
          'num_z_optimize_epochs': 100,
          'z_optimize_lr': 1e-3,
          'hidden_dim': 5, # default 32 ### to be divisible by nhead after concat with population features
          'post_hoc_hidden_dim': 2048, # default = hidden_dim
          'embedding_dim': 64, # 8 # default 8
          'layer_dim': 1, # default 1
          'window_len': 200, # default 200 ###
          'full_context': False, # default True ###
          'full_causal': False, # default False ###
          'context_backward': 1, # default -1 ###
          'context_forward': 0, # default -1 ###
          'attend_to_self': True, # default True ###
          'ckpt_path': None, # default None ###
          'use_ckpt_hparams': True, # default True,
          'post_hoc_only': False, # default False,
          'use_weighted_ce_loss': False # default False
}
